src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp

```diff
@@ -335,56 +393,78 @@ class brann_bronzebeard : public CreatureScript
             {
                 case ACTION_START_EVENT:
                     Start(false, true, ObjectGuid::Empty, 0, true, false);
+                    me->Yell("Time to get some answers! Let's get this show on the road!", LANG_UNIVERSAL);
```

Comment:
this text is also available as broadcast ID 30524 and 30561. I'm pretty sure using broadcast / database texts is highly preferred instead of declaring new text strings in the cpp files.

Can you please check if the text lines could be updated to use broadcast text instead? There's a broadcast editor / search tool included in Keira, too.